### PR TITLE
Fixed typos in README

### PR DESCRIPTION
**Pull request by rakshazi**
Hi,
I used [LanguageTool](https://languagetool.org) to fix typos in README

### CODE REVIEW

**Comment by spantaleev** [comment]
Looks like this destroys quite a lot of newlines/whitespace, which probably makes the README less readable in an editor.

**Comment by rakshazi** [comment]
Missed that moment

I'll update PR

**Comment by rakshazi** [comment]
@spantaleev done, newlines are back

File: README.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ Using this playbook, you can get the following services configured on your serve
 
 - (optional, default) an [Element](https://app.element.io/) ([formerly Riot](https://element.io/previously-riot)) web UI, which is configured to connect to your own Synapse server by default
 
-- (optional, default) an [ma1sd](https://github.com/ma1uta/ma1sd) Matrix Identity server
+- (optional, default) a [ma1sd](https://github.com/ma1uta/ma1sd) Matrix Identity server
 
 - (optional, default) an [Exim](https://www.exim.org/) mail server, through which all Matrix services send outgoing email (can be configured to relay through another SMTP server)
 
-- (optional, default) an [nginx](http://nginx.org/) web server, listening on ports 80 and 443 - standing in front of all the other services. Using your own webserver [is possible](docs/configuring-playbook-own-webserver.md)
+- (optional, default) a [nginx](http://nginx.org/) web server, listening on ports 80 and 443 - standing in front of all the other services. Using your own webserver [is possible](docs/configuring-playbook-own-webserver.md)
```
Comment (spantaleev) [inline README.md:36]: Given that this [is pronounced "engine-X"](https://en.wikipedia.org/wiki/Nginx), I'm guessing this one is a false-positive.

Not sure how `ma1sd` is meant to be pronounced. Perhaps "M A ONE S D", in which case *an* seems appropriate. Or is it more like "mazed"? :blush:

Comment (rakshazi) [reply README.md:36]: Agree on the first case, but literally have no idea about the second one :D

**Comment by spantaleev** [comment]
Thanks! :+1:

**Pull request merged**